tests/Unit/Crypt/AES/TestCase.php

```diff
@@ -61,6 +62,7 @@ public function continuousBufferCombos()
         foreach ($modes as $mode) {
             foreach ($plaintexts as $plaintext) {
                 foreach ($ivs as $iv) {
+                    if ($mode === BlockCipher::MODE_IGE) $iv .= strrev($iv);
```

Comment:
I suppose it doesn't matter as much in a unit test but, technically, this is a violation of the [PSR-2 coding standards](http://www.php-fig.org/psr/psr-2/#control-structures). That should read more like this:

```php
if ($mode === BlockCipher::MODE_IGE) {
    $iv.= strrev($iv);
}
```